src/test_suite/test_suite.ts

```diff
@@ -609,16 +610,21 @@ export class TestSuite {
             type: 'array',
         };
 
-        const ajv = new AJV();
+        const ajv = new AJV({ jsonPointers: true });
```

Comment:
Curious why you had to add jsonPointers: true here

Reply:
I'm not entirely sure, but it is required by the better ajv npm package

https://github.com/atlassian/better-ajv-errors#usage